sem_covid/entrypoints/notebooks/rdf_transformer/rml_mapper_executor.py

```diff
@@ -0,0 +1,55 @@
+from abc import ABC, abstractmethod
+import json
+import requests as req
```

Comment:
If we leave the full library name(i.e. requests instead of req), the code is more readable.